src/Handler.ts

```diff
@@ -31,6 +31,7 @@ export class Handler {
   constructor(options?: Options) {
     this.options = {...DEFAULT_OPTIONS, ...options};
     const elementId = this.options.id;
+    this.el = null!; // We create the tooltip element later, when we need it. This also has the advantage that if you have multiple charts, not multiple elements are created, as each chart will grab the existing one when its needed.
```

Comment:
How about we allow el to be null in the types. Then you can remove the `!` here.